README.md

```diff
@@ -1,35 +1,53 @@
-# AAVE Governance Forge Template
+# AAVE ETH Interest Rate Strategy Update
 
-A template for creating AAVE governance Proposal payload contracts.
+This repository contains the payload to update the AAVE V2 WETH Liquity Pool interest rate strategy.
 
-## Installation
+## Specification
 
-It requires [Foundry](https://github.com/gakonst/foundry) installed to run. You can find instructions here [Foundry installation](https://github.com/gakonst/foundry#installation).
+The Proposal Payload does the following:
 
-### GitHub template
+1. Sets the new interest rate strategy address.
 
-It's easiest to start a new project by clicking the ["Use this template"](https://github.com/llama-community/aave-governance-forge-template).
+The new interest rate strategy is deployed here: https://etherscan.io/address/0x2cbf7856f51660aae066afababf9c854fa6bd11f
 
```

Comment:
Would also add the new rate strategy parameters, or even better the table with the old vs new values